HLeithner/joomla-cms
# Add prepared statements for mod_related_items

### Summary of Changes
Updated SQL queries to prepared statements and made some cleanups around the queries.


### Testing Instructions
Use the module in all ways you can think of.

* create a frontend module
* create articles with the same metatags
* You should see related items based on the metatags matching

### Expected result
Nothing changed.
